style.css

```diff
@@ -401,6 +401,70 @@ body {
   flex-direction: column;
 }
 
+.overlay {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #191765;
+  overflow-x: hidden;
+  transition: 0.5s;
+  background-image: url(./bgimages/image-bg-1.png);
+  background-position: left top;
+  background-repeat: no-repeat, no-repeat;
+}
+
+.overlay-content {
+  position: relative;
+  top: 25%;
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.overlay a {
+  padding: 8px;
+  text-decoration: none;
+  margin-top: 15px;
+  font-size: 36px;
+  display: block;
+  transition: 0.3s;
+  text-align: left;
+  margin-left: 30px;
+  margin-right: 30px;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+}
+
+.overlay a:hover,
+.overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
```

Comment:
- [ ] Please reduce the size and right padding of the close(X) button and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168053885-31a38cc5-7f5d-4d20-86ca-f3867f2dcd48.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168053665-e001bcad-a8fb-4de2-93e8-b35769dd9abd.png)